structuredefinitions/England-Patient.xml

```diff
@@ -156,12 +156,6 @@
       <path value="Patient.address.use" />
       <short value="The following values are allowed: home, work, temp, billing" />
     </element>
-    <element id="Patient.address.line">
```

Comment:
Any reason to remove Patient.address.line and Patient.multipleBirth[x] elements?